polyfills/Symbol/config.json

```diff
@@ -8,7 +8,7 @@
 		"safari": "<9",
 		"chrome": "< 50",
 		"firefox": "< 40",
-		"ios_saf": "<9",
+		"ios_saf": "<10",
```

Comment:
Seems we are serving it to ios saf 10 -- https://polyfill.io/v2/polyfill.js?features=Symbol&ua=ios_saf/10&rum=1